rustler/build.rs

```diff
@@ -826,7 +833,7 @@ fn build_api(b: &mut dyn ApiBuilder, opts: &GenerateOptions) {
         // Func("int",  "enif_ioq_peek_head",        "ErlNifEnv *env, ErlNifIOQueue *q, size_t *size, ERL_NIF_TERM *head"),
         // Func("char*, "enif_mutex_name",           "ErlNifMutex*"),
         // Func("char*, "enif_cond_name",            "ErlNifCond*"),
-        // Func("char*, "enif_rwlock_name",          "ErlNifRWLock*"),
+        // Func("char*, "enif_rwlock_name",          "ErlNifRWLock*"),buid
```

Comment:
What's this?

Reply:
https://github.com/user-attachments/assets/1df7ecb1-7bbc-4bd7-8498-93e0bd53f46d

I'll fix it :)